site/docs/master/install-overview.md

```diff
@@ -63,6 +63,80 @@ kubectl delete namespace/velero clusterrolebinding/velero
 kubectl delete crds -l component=velero
 ```
 
+## Installing with the Helm chart
+
+When installing using the Helm chart, the provider's credential information will need to be appended into your `values.yaml` file.
```

Comment:
Helm 2.10+ has a `--set-file` flag which would allow us to simply do `helm install --set-file credentials.secretContents.cloud=./credentials-velero` (https://github.com/helm/helm/pull/3758)